pkg/tar/tar.go

```diff
@@ -215,6 +222,22 @@ func (t *stiTar) ExtractTarStream(dir string, reader io.Reader) error {
 	}
 }
 
+func extractLink(dir string, header *tar.Header, tarReader io.Reader) error {
+	dest := filepath.Join(dir, header.Name)
+	source := header.Linkname
+
+	glog.V(3).Infof("Creating symbolic link from %s to %s", dest, source)
+
+	if err := os.Symlink(source, dest); err != nil {
```

Comment:
I don't know why we had these glogs here, but just noting that if there's an error, we did not print "Done with...", while if we use defer that way the message would be printed unconditionally.